code/modules/antagonists/abductor/machinery/camera.dm

```diff
@@ -59,16 +59,24 @@
 	return HAS_TRAIT(H, TRAIT_ABDUCTOR_SCIENTIST_TRAINING)
 
 /datum/action/innate/teleport_in
+///Is the amount of time required between uses
+	var/abductor_pad_cooldown = 8 SECONDS
+///Is used to run the timer for the built-in cool down, and will make the action fail if set
+	var/on_cooldown
 	name = "Send To"
 	icon_icon = 'icons/mob/actions/actions_minor_antag.dmi'
 	button_icon_state = "beam_down"
 
 /datum/action/innate/teleport_in/Activate()
 	if(!target || !iscarbon(owner))
 		return
+	if(on_cooldown)
+		to_chat(owner, "<span class='warning'>You must wait [timeleft(on_cooldown)*0.1] seconds to use the [target] again!</span>")
+		return
 	var/mob/living/carbon/human/C = owner
 	var/mob/camera/aiEye/remote/remote_eye = C.remote_control
 	var/obj/machinery/abductor/pad/P = target
+	on_cooldown = addtimer(VARSET_CALLBACK(src, on_cooldown, null), abductor_pad_cooldown , TIMER_STOPPABLE)
```

Comment:
curious, do timers have a greater performance cost vs just setting `on_cooldown = world_time + abductor_pad_cooldown` and then comparing world.time >= on_cooldown?

Reply:
I'm too inexperienced and uninformed to answer that.

Reply:
Needs TIMER_UNIQUE as well

Reply:
`change it to world_time + adbuctor_pad_cooldown` instead please.